src/components/JobDescriptionModal/index.jsx

```diff
@@ -0,0 +1,87 @@
+import TypeBadge from "../TypeBadge/index.jsx";
+import Salary from "../Salary/index.jsx";
+import SkillsBadge from "../SkillsBadge/index.jsx";
+import {useEffect, useState} from "react";
+
+const JobDescriptionModal = ({id}) => {
+
+    const [jobInfo, setJobInfo] = useState([])
+
+    useEffect(() => {
+        const getJobInfo = async () => {
+            const response = await fetch(`http://0.0.0.0:8080/jobs/${id}`)
+            const data = await response.json()
+            setJobInfo(data)
+        }
+        getJobInfo()
+    }, [id])
+
+    return (
+        <div className={'modal'} id={'jobDescription'}>
+            <div className={'modal-dialog modal-lg'}>
+                <div className={'modal-content'}>
+                    <div className={'modal-header d-flex justify-content-between align-items-center'}>
+                        <p className={'text-center m-0'}>{jobInfo.job_title} - {jobInfo.company}</p>
+                        <button type={'button'} className={'btn btn-sm btn-outline-info border-0'}
+                                data-bs-dismiss={'modal'}>ⅹ
```

Comment:
learnt from trainer review, anything that is a string, you take off {}

Reply:
Implemented.